src/main/java/com/dropbox/core/android/Auth.java

```diff
@@ -3,6 +3,11 @@
 import android.app.Activity;
```

Comment:
I'm looking for feedback in Auth.java and AuthActivity.java, where I added support in Short Live Token and PKCE in android.